website/content/en/preview/provisioner.md

```diff
@@ -149,13 +149,40 @@ Karpenter prioritizes Spot offerings if the provisioner allows Spot and on-deman
 
 ## spec.kubeletConfiguration
 
-Karpenter provides the ability to specify a few additional Kubelet args. These are all optional and provide support for
-additional customization and use cases. Adjust these only if you know you need to do so.
+Karpenter provides the ability to specify a few additional [Kubelet args](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration). These are all optional and provide support for additional customization and use cases. Adjust these only if you know you need to do so.
 
 ```yaml
 spec:
   kubeletConfiguration:
     clusterDNS: ["10.0.1.100"]
+  eventRecordQPS: 5
+  eventBurst: 10
```

Comment:
I think the spacing is incorrect for these - it should be on the same level as `clusterDNS`